src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -87,6 +91,64 @@
     /** The String "(null)". */
     private static final String NULL_STRING = "(null)";
 
+    /**
+     * Checks in which violation message is not specified in input file and have more than
+     * one violation message key.
```

Comment:
please create new issue to resolve  all this suppressions.
and reference this issue here as "until #xxxxxx"

Reply:
Done